src/pages/Intents/CustomIntentsList.js

```diff
@@ -0,0 +1,52 @@
+import { CircularLoader } from '@dhis2/ui'
+import isEqual from 'lodash/isEqual'
+import PropTypes from 'prop-types'
+import React, { useEffect, useMemo } from 'react'
+import TableActions from '../../components/TableActions'
+import NewIntent from './NewIntent'
+
+// TODO: create function to model the data that will be save in Datastore, get Data elements and attributes
+const CustomIntentsList = ({ settings, handleSettings, disable }) => {
+    const loading = false
+    const rows = settings
+    const initialRows = useMemo(() => {
+        return settings
+    }, [settings])
+
+    useEffect(() => {
+        if (rows && initialRows && !isEqual(rows, initialRows)) {
+            handleSettings(rows)
```

Comment:
I know this is a placeholder for now, but I assume that for when the add/update/delete functionality edit gets added, we'd want to do it with functions passed down from CustomIntents.js and not having an effect firing based on change of state from here? (At the moment, `rows` also comes from the parent component, so there's also no actual component for the state to respond to anyway)